Cargo.toml

```diff
@@ -58,9 +69,4 @@ env_logger = "0.9.0"
 tonic-build = {version = "0.9.2",features = ["prost"]}
 
 [workspace]
```

Comment:
I _think_ we no longer need the exclude and can get rid of the `[workspace]` section entirely.